server/src/test/java/org/spine3/server/storage/Given.java

```diff
@@ -0,0 +1,404 @@
+            return StartProject.newBuilder()
+                               .setProjectId(id)
+                               .build();
+        }
+    }
+
+    /* package */ static class Event {
+
+        private static final ProjectId PROJECT_ID = AggregateId.newProjectId();
+
+        private Event() {
+        }
+
+        /**
+         * Creates a new {@link org.spine3.base.Event} with default properties.
+         */
+        public static org.spine3.base.Event projectCreatedEvent() {
+            return projectCreatedEvent(PROJECT_ID);
+        }
+
+        /**
+         * Creates a new {@link org.spine3.base.Event} with the given projectId.
+         */
+        public static org.spine3.base.Event projectCreatedEvent(ProjectId projectId) {
+            return projectCreatedEvent(projectId, createEventContext(projectId));
+        }
+
+        /**
+         * Creates a new {@link org.spine3.base.Event} with the given projectId and eventContext.
+         */
+        public static org.spine3.base.Event projectCreatedEvent(ProjectId projectId, EventContext eventContext) {
+            final ProjectCreated eventMessage = EventMessage.projectCreatedMsg(projectId);
+            final org.spine3.base.Event.Builder builder = org.spine3.base.Event.newBuilder()
+                                                                               .setContext(eventContext)
+                                                                               .setMessage(toAny(eventMessage));
+            return builder.build();
+        }
+
+        /**
+         * Creates a new {@link org.spine3.base.Event} with the given projectId and eventContext.
+         */
+        public static org.spine3.base.Event taskAddedEvent(ProjectId projectId, EventContext eventContext) {
+            final TaskAdded eventMessage = EventMessage.taskAddedMsg(projectId);
+            final org.spine3.base.Event.Builder builder = org.spine3.base.Event.newBuilder()
+                                                                               .setContext(eventContext)
+                                                                               .setMessage(toAny(eventMessage));
+            return builder.build();
+        }
+
+        /**
+         * Creates a new {@link org.spine3.base.Event} with the given projectId and eventContext.
+         */
+        public static org.spine3.base.Event projectStartedEvent(ProjectId projectId, EventContext eventContext) {
+            final ProjectStarted eventMessage = EventMessage.projectStartedMsg(projectId);
+            final org.spine3.base.Event.Builder builder = org.spine3.base.Event.newBuilder()
+                                                                               .setContext(eventContext)
+                                                                               .setMessage(toAny(eventMessage));
+            return builder.build();
+        }
+
+    }
+
+    /* package */ static class RecordStorage {
```

Comment:
Pls rename to `AggregateStorageRecord`, and `newAggregateStorageRecord` methods to `create`.